tests/dist-check.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Alright, then you need to upgrade `.pre-commit-config.yaml` and exclude this file from test name checker :)